packages/odyssey-icons/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@okta/odyssey-icons",
+  "version": "0.3.0",
+  "description": "Odyssey's icon library",
```

Comment:
make this package private and change it to public once its ready to be consumed.
Set ```"private": true```

Reply:
By the next release, we'll want it to be publicly available so that our non-okta-core teams can start utilizing Icon. I can swap this to `private` now, but it won't make a difference until we publish the package, right?

Reply:
This is more of a best practice suggestion.

If we don't mark it as `private`, it'll still go through our artifact promotion process. By marking it as such, when `lerna publish` gets triggered, it'll ignore this package.